CondTools/Geometry/test/geometrywriter.py

```diff
@@ -18,11 +18,6 @@
                                            ZIP = cms.untracked.bool(True)
                                            )
 
-process.TrackerGeometricDetESModule = cms.ESProducer( "TrackerGeometricDetESModule",
-                                                      fromDDD = cms.bool( True )
```

Comment:
I don't understand.
Isn't the auto-generated one using:
```
fromDDD = cms.bool( False )
```
https://github.com/cms-sw/cmssw/blob/e066be0574d4aa3957ea34b307f9f3256add2a52/Geometry/TrackerNumberingBuilder/plugins/TrackerGeometricDetESModule.cc#L52

shouldn't this be `esprefered` as @vargasa said to preserve the behaviour?

Reply:
@mmusich - it would be if it were `trackerNumberingGeometryDB` and not  `trackerNumberingGeometry` included. `GeometryExtended2021_cff` defines a transient scenario. @cvuosalo's solution is correct.

Reply:
I tested that what you say is true:
```
cmsrel CMSSW_11_3_X_2020-12-01-230
cd CMSSW_11_3_X_2020-12-01-2300/src/
git cms-addpkg CondTools/Geometry/
cd CondTools/Geometry/test/
cmsenv
edmConfigDump geometrywriter.py > dump.py
more dump.py | grep -A 4 TrackerGeometricDetESModule
```

gives:
```python 
process.TrackerGeometricDetESModule = cms.ESProducer("TrackerGeometricDetESModule",
    fromDDD = cms.bool(True)
)
--
process.trackerNumberingGeometry = cms.ESProducer("TrackerGeometricDetESModule",
    appendToDataLabel = cms.string(''),
    fromDD4hep = cms.bool(False),
    fromDDD = cms.bool(True)
)
```

incidentally when one uses `edmConfigDump` there is this warning:
```
/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc900/cms/cmssw/CMSSW_11_3_X_2020-12-01-2300/python/Configuration/Geometry/GeometryExtended_cff.py:2: UserWarning: ATTENTION: This job uses Run 1 Geometry.
  warnings.warn('ATTENTION: This job uses Run 1 Geometry.')

```

Reply:
The warning is correct - `geometrywriter.py` picks up the Run 1 geometry scenario: `Configuration.Geometry.GeometryExtended_cff`